### PR TITLE
Fix CodeMirror RTL issues in Query modules

### LINKED ISSUE #3668: RTL issue in the Query pages
This issue is similar to #3652, where the query code field shouldn't be aligned to the right, PR should fix both of them ..

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #3668 and #3652

### CODE REVIEW

**Comment by hishamco** [comment]
@aghili371 please try this PR, so you can confirm that it solves your issue too

**Comment by sebastienros** [comment]
dont't forget to rebase before submitting your PRs

**Comment by hishamco** [comment]
> dont't forget to rebase before submitting your PRs

Sure I always remember that .. thanks for the reminder again, I just facing some Internet issues these few days, that's why I delay some times between commits :)

**Comment by hishamco** [comment]
Query

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml [hunk 1]
```diff
@@ -10,6 +10,6 @@
 
 <fieldset class="form-group">
     <label asp-for="Schema">@T["Schema"]</label>
-    <textarea asp-for="Schema" rows="12" class="form-control"></textarea>
+    <textarea asp-for="Schema" rows="12" class="form-control" style="text-align:left"></textarea>
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml]: Is that necessary since you changed it in the codemirror css?

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml]: Yep, this is for Schema textarea I made it inline style to enforce this field to align to left even in RTL

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml]: ok

**Pull request merged**